debs/SPECS/3.7.0/wazuh-agent/debian/postrm

```diff
@@ -10,12 +10,26 @@ WAZUH_TMP_DIR="${DIR}/tmp/installation_files"
 case "$1" in
     remove|failed-upgrade|abort-install|abort-upgrade|disappear)
     
-	if getent passwd | grep -q "^ossec" ; then
-		deluser ossec > /dev/null 2>&1
-	fi
-	if getent group | grep -q "^ossec" ; then
-		delgroup ossec > /dev/null 2>&1
-	fi
+    # If the host uses systemd, disable the service and
+    # remove it from /etc/systemd/system/
+    if [ -d /run/systemd/system ] ; then
+        systemctl disable wazuh-agent > /dev/null 2>&1
+        rm -f /etc/systemd/system/wazuh-agent.service
+        update-rc.d -f wazuh-agent remove
+        rm -f /etc/init.d/wazuh-agent
+        systemctl daemon-reload
```

Comment:
In comparison with the RPM package, we need to ensure the order of the commands in terms of functionality and homogeneity. Please review it and we can decide if first "daemon-reload" or "reset-failed"